joomla/joomla-cms
# [4.0] ES6 please, com modules

Pull Request for Issue # .

### Summary of Changes
Com_modules script to ES6


### Testing Instructions
- Edit an article and try to insert a module through the XTD button (position or title)
- Edit a menu and check the last tab. Try to alter something in any module, the changes should reflect on the menu itself
- Try to edit a module 

### Expected result



### Actual result



### Documentation Changes Required
